src/gui/qjacktrip.ui

```diff
@@ -1408,7 +1408,12 @@ for better quality at the expense of latency.</string>
           </item>
           <item>
            <property name="text">
-            <string>3 (experimental)</string>
+            <string>3 (experimental - in own thread) </string>
+           </property>
+          </item>
+          <item>
+           <property name="text">
+            <string>4 (experimental - in callback)</string>
```

Comment:
This should be the only change needed in the .ui file. (Not sure how some of the others crept in.) If you're able to revert all but this one that would be great.

Reply:
those were side effects of me editing using designer, please check if I reverted  'em ok
lesson learned!

Reply:
I use designer as well, but usually it behaves itself a bit better. There's just another couple at line 1909 and 1989. (Otherwise it's looking good.)

Reply:
I wonder if all these nits might be due to display differences or me touching the designer objects awkwardly, I missed those last ones down there at the bottom of meld -- fixed now

Reply:
I tested the installer artifact under win10 and it works as intended